packages/talos_common_utils/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
👍  Thanks for creating the `talos_common_utils` I think with so many sub projects in the talos umbrella. This is going to be really useful. 

I have been thinking on similar lines, but mostly for `models`. Just haven't had the push yet.